src/marching_cubes.cpp

```diff
@@ -502,14 +503,6 @@ void MarchingCubes::cleanup()
   MyPage<surfint> *csurfs = grid->csurfs;
   int nglocal = grid->nlocal;
 
-  // DEBUG
```

Comment:
These debugging statements can be useful, please restore.

Reply:
Added back these lines.